src/main/java/de/tum/cit/aet/artemis/communication/service/notifications/MailService.java

```diff
@@ -224,6 +239,7 @@ public void sendNotification(Notification notification, Set<User> users, Object
      * @param user                who should be contacted
      * @param notificationSubject that is used to provide further information (e.g. exercise, attachment, post, etc.)
      */
+
```

Comment:
Remove this new line

Reply:
Thank you for the review and the heads up!
I fixed the server-style issues and removed the new line.